### PR TITLE
add_test_subctl_benchmark

### PR DESCRIPTION
**Pull request by pinikomarov**
This CI Test will test if subctl benchmarks are working fine

### CODE REVIEW

File: setup_subm.sh [hunk 1]
```diff
@@ -3581,6 +3581,22 @@ function test_project_e2e_with_go() {
 
 # ------------------------------------------
 
+function test_subctl_benchmarks() {
+  PROMPT "Testing subctl benchmark: latency and throughput tests"
+  trap_to_debug_commands;
+
+  export "KUBECONFIG=${KUBECONF_CLUSTER_A}"
+
+  export "KUBECONFIG=${KUBECONF_CLUSTER_B}"
```
Comment (manosnoam) [inline setup_subm.sh:3590]: If we need both clusters accessible, please change it to: 
`export KUBECONFIG="${KUBECONF_CLUSTER_A}:${KUBECONF_CLUSTER_B}"`

File: setup_subm.sh [hunk 2]
```diff
@@ -3581,6 +3581,22 @@ function test_project_e2e_with_go() {
...
+
+  subctl benchmark latency ${KUBECONF_CLUSTER_A} ${KUBECONF_CLUSTER_B}
+
+  subctl benchmark throughput ${KUBECONF_CLUSTER_A} ${KUBECONF_CLUSTER_B}
```
Comment (manosnoam) [inline setup_subm.sh:3594]: When referring here to "${KUBECONF_CLUSTER_A}" and  "${KUBECONF_CLUSTER_B}"
did you mean the cluster name/id ? 

e.g.
`subctl benchmark latency ${CLUSTER_A_NAME} ${CLUSTER_B_NAME}`

if it's the kubeconfig file (and not cluster name/id), then the previous command of exporting kubeconfig seems to be redundant.

**Review by manosnoam — CHANGES_REQUESTED** [review]

File: setup_subm.sh (hunk 2 above)
Comment (pinikomarov) [reply setup_subm.sh:3590]: correct , the previous export statements are redundant, I'll change that..

**Review by pinikomarov — COMMENTED** [review]
removed the "export" lines

**Review by manosnoam — CHANGES_REQUESTED** [review]
You did not call the test, but only defined it.
Can you call it on line 4164 
https://github.com/manosnoam/ocp-multi-cluster-tester/blob/cb0fdcfb6a6a1f2c68c368f4b30404ef360e54a4/setup_subm.sh#L4164

`${junit_cmd} test_subctl_benchmarks`

**Pull request closed without merging**